### ISSUE TITLE
"Mixing tabs and non-tab-content is not possible" when providing slots

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/nextcloud-vue#733]
In Talk we provide tabs for the sidebar and also provide a template for the `secondary-actions` slot, but this throws a red warning in the console:

```
 [Vue warn]: Mixing tabs and non-tab-content is not possible. v-tooltip.esm.js:3980
    warn v-tooltip.esm.js:3980
    updateTabs AppSidebar.js:12
    mounted AppNavigationCounter.js:1115
    VueJS 13
    js talk.js:104855
    Webpack 3
```

Removing the slot works:
https://github.com/nextcloud/spreed/blob/f9881d670a28832d0f4da541ed91d26230453c07/src/components/Sidebar/Sidebar.vue#L29-L47

But I guess it should just not be that strict on checking and basically allow templates here that are for slots:
https://github.com/nextcloud/nextcloud-vue/blob/master/src/utils/ValidateChildren.js#L39

**Comment by skjnldsv** [comment]
Hum, this should not be an issue.
Are you sure this is coming from the template? :thinking:

**Comment by raimund-schluessler** [comment]
Fix is in #1281.
